src/widget/form/chatform.cpp

```diff
@@ -783,9 +784,11 @@ void ChatForm::onScreenshotClicked()
 
 void ChatForm::doScreenshot()
 {
-    ScreenshotGrabber* screenshotGrabber = new ScreenshotGrabber(this);
-    connect(screenshotGrabber, &ScreenshotGrabber::screenshotTaken, this, &ChatForm::onScreenshotTaken);
+    screenshotGrabber = new ScreenshotGrabber(this);
+    connect(screenshotGrabber, &ScreenshotGrabber::screenshotTaken,
+            this, &ChatForm::onScreenshotTaken, Qt::QueuedConnection);
```

Comment:
Yeah, I think I know what's going on here. I remember that grabber code and it's greatly f***ed up… :smile: Anyway, strange things happen, because the grabber `window' member executes code after (!) the object got deleted -> memory corruption :boom:. By forcing a queued connection, the slot (event) is executed after the window is closed - but it's a pure coincidence and even f***s up the code more. Would you mind cleaning this up, by copying the rect to another var and move the "emit" to the end inside `ScreenshotGrabber::acceptRegion`?